src/sentry/search/events/constants.py

```diff
@@ -191,3 +191,22 @@ class ThresholdDict(TypedDict):
     "measurements.ttfb.requesttime",
     "transaction.duration",
 ]
+# So we can dry run some queries to see how often they'd be compatible
+DRY_RUN_COLUMNS = {
```

Comment:
Are all these columns available for all `distribution / set / counter` ?

Reply:
No, but the intent is the query joining that happens in run_query _should_ handle that.
Regardless though I do want to point out that this will have a small false positivity rate, eg. a filter like
`tag[transaction.op]:timestamp`
will be marked as compatible even though it is not